packages/styles/src/placeholders/_button.scss

```diff
@@ -23,3 +23,12 @@
     color: rgba(var(--post-contrast-color-rgb), 0.6);
   }
 }
+
+%btn-transparent-background {
```

Comment:
The text color an :hover text color could probably be part of this placeholder as well.